mobile/src/main/java/rs/readahead/washington/mobile/views/activity/viewer/VideoViewerActivity.kt

```diff
@@ -504,7 +489,7 @@ class VideoViewerActivity : BaseLockActivity(), StyledPlayerControlView.Visibili
         options[1] = R.string.verification_share_select_media_and_verification
         options[0] = R.string.verification_share_select_only_media
         showRadioListOptionsSheet(supportFragmentManager,
```

Comment:
Use a more concise way to initialize the options map: Instead of using LinkedHashMap and explicitly adding each option, maybe ? 
```
val options = mapOf(
        R.string.verification_share_select_media_and_verification to R.string.verification_share_select_media_and_verification,
        R.string.verification_share_select_only_media to R.string.verification_share_select_only_media
    )
```

Reply:
apply it for the rest of the code if applicable!